data/org.freedesktop.portal.GlobalShortcuts.xml

```diff
@@ -199,6 +221,24 @@
         @shortcuts: The different shortcuts that have been registered. See org.freedesktop.portal.GlobalShortcuts.
 
         Indicates that the information associated with some of the shortcuts has changed.
+
+        Each element of the @shortcuts array is a tuple composed of a shortcut id,
+        and a vardict with the following keys:
+        <variablelist>
+          <varlistentry>
+            <term>description s</term>
+            <listitem><para>
+              User-readable text describing what the shortcut does.
+            </para></listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>trigger_description s</term>
+            <listitem><para>
+              User-readable text describing how to trigger the shortcut for the
+              client to render.
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
```

Comment:
Can't we just refer to the list in BindShortcuts?

Reply:
Sure. Done.